cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -125,6 +125,8 @@ protected void createContent(final Bundle savedInstanceState) {
             "--enable-features=LogJsConsoleMessages",
             // Disable rescaling Webpage.
             "--force-device-scale-factor=1",
+            // Enable Cobalt-specific JS attributes/methods.
+            "--enable-blink-features=H5vccAudioConnectors,SourceBufferWriteHead",
```

Comment:
@yell0wd0g For Cobalt-specific idl features, media team try to use blink runtime features for these attributes, please help review if this is the right way for us.

Reply:
Also, is it possible to do this on platforms that run the Cobalt binary directly, like Linux? We can add this param for a few platforms like Android, but I'm not sure if we'll need to run the Linux Cobalt binary with this param separately.

Reply:
For Linux, see https://github.com/youtube/cobalt/blob/main/cobalt/cobalt.cc.